src/component/Removebook.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/Books/Books';
```

Comment:
- [ ] Kindly enable linters here to ensure the standard is being met.

- [ ] App does not run due to Line 4 here as import names do not match file names. We should have this:

  ```
  import { removeBook } from '../redux/books/books';
  ```

Reply:
fixed